app/models/container_node.rb

```diff
@@ -1,5 +1,5 @@
 class ContainerNode < ApplicationRecord
   belongs_to :tenant
   belongs_to :source
-  has_many   :container_groups, :dependent => :nullify
```

Comment:
@agrare changed this to delete, I think pod can't live without a node to run on?